std/process.d

```diff
@@ -1160,7 +1169,7 @@ private Pid spawnProcessImpl(scope const(char[])[] args,
                     errorMsg = "getrlimit failed";
                     break;
                 case InternalError.exec:
-                    errorMsg = "Failed to execute program";
+                    errorMsg = "Failed to execute '" ~ cast(string) name ~ "'";
```

Comment:
Because POSIX file names may contain characters such as newlines or quote characters, may I suggest
```suggestion
                    errorMsg = "Failed to execute " ~ escapeShellFileName(name);
```